css/pysangamam.css

```diff
@@ -208,3 +209,8 @@ svg.icon-link {
     text-align: center;
   }
 }
+
+a{
+  color : #218c74;
+  text-decoration: None;
+}
```

Comment:
You will also need to style the pseudo classes for the anchor tag to make this work properly. As in, add the styles for a:link, a:visited, a:active, a:hover.